### ISSUE TITLE
Document "good practice" for jaw-set IOC

### ISSUE THREAD

**Issue by kjwoodsISIS** [kjwoodsISIS/IBEX#189]
As a developer I want to document the lessons learned during the implementation of the jaw-set IOC, so that I can apply "good practice"  to the implementation of other business logic IOCs.

'''Acceptance Criteria'''
1. A document recording the lessons learned during the implementation of the jaw-set IOC (see ticket:179).
1. Document published on Wiki or ICP Discussions site

'''Notes'''
1. None.

**Comment by kjwoodsISIS** [comment]
_Imported from trac issue 191.  Created by skn09965 on 2013-08-29T14:37:25, last modified: 2013-10-23T12:40:02_

**Comment by kjwoodsISIS** [comment]
_Trac comment by kvlb23 on 2013-09-26 16:27:41:_

As stated in the meeting I took on this story today to ensure that a good amount was undertaken whilst still fresh in my mind. I have actually created two wiki pages, LogicLessonsLearned and LogicGoodPractice which highlights some of what came out of the lessons learned. There are I am sure items to add to these documents, but this is a good portion of what I came across and what is necessary to undertake this work from here on. I shall reassign it for now, and close the task tomorrow when others have had a chance to look at the pages and say if there is anything they would like to see differently in relation to them.
